drivers/gpio/Kconfig.sx1509b

```diff
@@ -14,6 +14,7 @@ menuconfig GPIO_SX1509B
 
 config GPIO_SX1509B_INIT_PRIORITY
 	int "Init priority"
+	depends on GPIO_SX1509B
```

Comment:
Could put all these symbols (or the `source` of `Kconfig.sx1509b`) within a `if GPIO_SX1509B` block as well, which is equivalent.

Reply:
(I'm about to add an include path display to `menuconfig` and the generated documentation btw. That will make it easier to see how files get included. I usually just grep.)

Reply:
Urr... just noticed that `GPIO_SX1509B` is defined at the top of the file, so disregard that. Could use an `if` still. :)

Reply:
So to be clear, just wrap the config items with:
```
if GPIO_SX1509B
...
endif # GPIO_SX1509B
```

Reply:
Doesn't really need the "depends on" then does it?

Reply:
Yep, and the `depends on` won't be needed then.

`if X` is just syntactic sugar for adding `depends on x` to every item within the `if` block.